ee/zentral/contrib/intune/models.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from django.db import models
+from django.db.models import F
+from django.urls import reverse
+
+from zentral.core.secret_engines import decrypt_str, encrypt_str, rewrap
+
+
+logger = logging.getLogger("zentral.contrib.intune.models")
+
+
+class Tenant(models.Model):
+    business_unit = models.ForeignKey("inventory.BusinessUnit", on_delete=models.PROTECT)
+    name = models.CharField(max_length=256, unique=True)
+    description = models.TextField(blank=True, null=True)
+    # Authentication
+    tenant_id = models.CharField(max_length=256, help_text="The microsoft Azure Tenant ID")
```

Comment:
I think this field should have a unique=True constraint. Sorry that I missed it before.

Reply:
Thought about it. I'll add it.
I'm not happy with having name as `unique=True` but was on the specs.